assets/texts/en/Changelog/next.md

```diff
@@ -1,3 +1,5 @@
 * **New:** The QR code now shows a warning if you try to enter too much text. ([#168](https://github.com/rugk/offline-qr-code/pull/168), thanks [@ongspxm](https://github.com/ongspxm))
+* **Improved:** The options page has been revamped and fits better into the style of the Photon options. ([#170](https://github.com/rugk/offline-qr-code/pull/170), thanks [@ongspxm](https://github.com/ongspxm))
```

Comment:
It seems you copied & pasted the changes here.
Note for the next time: Better merge my updated master into your branch, i.e. `git merge master` (after you have updated your master to get to the same state as my master.

Anyway, does not matter right now. :smile:

Reply:
Yes, I did :) Alright captain. I will follow the rules next time :+1:

Reply:
Well, it's just a best practice for git in general… :smile: